app/exporter/models/user.py

```diff
@@ -0,0 +1,16 @@
+"""
+    This is the model class for an user.
+"""
+
+from flask import abort
+
+
+class User():
```

Comment:
Not sure why you need a separate class for a user with a single field, doesn't it over-complicate things?